tests/ansible_collections/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- hosts: all
+  roles:
+
+    - role: oracle-linux-specific
+      when: ansible_facts['distribution'] == "OracleLinux"
+      tags:
+        - oracle
```

Comment:
What's the purpose of the tags?

Reply:
Similar to defaults in roles. Agree let's remove it unless we'll feel a need in the possibility to reuse roles for maunall runs. The tags allow running certain scenarios, i.e.
ansible-playbook --tags packaging will run only roles in sequence with tag packaging.